public/app/plugins/datasource/loki/querybuilder/components/UnwrapParamEditor.tsx

```diff
@@ -53,15 +54,17 @@ export function UnwrapParamEditor({
 
 async function loadUnwrapOptions(
   query: LokiVisualQuery,
-  datasource: LokiDatasource
+  datasource: LokiDatasource,
+  timeRange?: TimeRange
```

Comment:
```suggestion
  timeRange = getDefaultTimeRange()
```

Reply:
Fixed in https://github.com/grafana/grafana/pull/78642/commits/acb281e84d70dfbf84feed52713677e5024f4091